source/simulator/stokes_matrix_free.cc

```diff
@@ -801,14 +802,78 @@ namespace aspect
       }
   }
 
+
+
+  template <int dim, int degree_v, typename number>
+  void
+  MatrixFreeStokesOperators::StokesOperator<dim, degree_v, number>
+  ::local_apply_face(const dealii::MatrixFree<dim, number> &,
+                     dealii::LinearAlgebra::distributed::BlockVector<number> &,
+                     const dealii::LinearAlgebra::distributed::BlockVector<number> &,
+                     const std::pair<unsigned int, unsigned int> &) const
+  {
+  }
+
+
+
+  template <int dim, int degree_v, typename number>
+  void
+  MatrixFreeStokesOperators::StokesOperator<dim, degree_v, number>
+  ::local_apply_boundary_face(const dealii::MatrixFree<dim, number> &data,
+                              dealii::LinearAlgebra::distributed::BlockVector<number> &dst,
+                              const dealii::LinearAlgebra::distributed::BlockVector<number> &src,
+                              const std::pair<unsigned int, unsigned int> &face_range) const
+  {
+    FEFaceEvaluation<dim, degree_v, degree_v + 1, dim, number> velocity(data,
+                                                                        true, /*is_interior_face*/);
```

Comment:
Shouldn't this be false? Also remove ,

Reply:
is interior face of the cell, I should have been more specific.